src/TableOfContents/index.js

```diff
@@ -34,14 +34,14 @@ const TableOfContents = ({ tableOfContents, ...props }) => {
       marginY='size-400'
       {...props}
     >
-      <h4
+      <span
```

Comment:
What is the problem with h4 ?

Reply:
The headers are supposed to go down one step at a time so a random h4 in there reports a minor a11y issue.